src/Microsoft.DotNet.Interactive.HttpRequestParser/Reference/DynamicVariable.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.DotNet.Interactive.HttpRequest.Reference;
+
+internal record DynamicVariable(Regex Expression, Func<HttpDocumentSnapshot, string, MatchCollection, string> InvokeHandler);
```

Comment:
Are the files under the `Reference` folder needed here?

If we can remove them and the tests still pass and things compile, now is probably a good time to delete them.

Reply:
Yes, I agree we can remove them. They were added so that we could easily see what the old parser was doing with its regexes - but we could just as easily look at the original code in the other repo for this...